composer.json

```diff
@@ -28,7 +28,7 @@
         "squizlabs/php_codesniffer": "~3.5.3",
         "slevomat/coding-standard": "^6.0",
         "dealerdirect/phpcodesniffer-composer-installer": "^0.5 || ^0.7",
-        "phpstan/phpstan": "^0.12",
+        "phpstan/phpstan": "^0.12.65",
```

Comment:
This is really needed? It should not, IMHO

Reply:
There was an issue with low versions of PhpStan on PHP 8.0 and 8.1. See the CI on 12587a6. I didn't really want to debug this, so I've felt that upgrading to at least a version from the beginning of the year.

If that's ok with you, mark the comment as resolved and it will be merged automatically 🙂